test/common_quantization.py

```diff
@@ -49,6 +51,12 @@ def test_only_train_fn(model, train_data, loss_fn=_default_loss_fn):
             correct += (predicted == target).sum().item()
     return train_loss, correct, total
 
+def convert_dynamic(module):
+    convert(module, DEFAULT_DYNAMIC_MODULE_MAPPING)
+
+def prepare_dynamic(model, qconfig_dict=None):
+    propagate_qconfig(model, qconfig_dict)
```

Comment:
I guess qconfig_dict is already gone

Reply:
This is a utility function only for the testing purpose. Do you mean that we should remove the `qconfig_dict` in this function call (`propagate_qconfig` still has `qconfig_dict` argument: https://github.com/pytorch/pytorch/blob/master/torch/quantization/quantize.py#L33)?

For dynamic quantization, we will still keep qconfig_dict (discussed with @raghuramank100 , @jerryzh168 ). Will submit a Diff to set default `qconfig_dict` to correspond to `torch.nn.quantized.dynamic.Linear` (after PR #23212), as you and @raghuramank100 suggested.